### ISSUE TITLE
Collaboration: Send and recieve rate limited by frame speed

### ISSUE THREAD

**Issue by karl-zylinski-ourmachinery** [OurMachinery/themachinery-public#1097]
- **Version:** Git Master Branch
- **OS:** Windows

When syncing a big project me and @tobias-ourmachinery noticed that the send / receive rate is limited because it happens as part of the editor UI update. We think it should happen on a separate thread. We could get no more than 25-45 mbit/s.

**Comment by Tisten** [comment]
Good point. Our headless server always runs full speed and doesn't do any rendering so I haven't noticed this much.
